presto-oracle/src/test/java/com/facebook/presto/plugin/oracle/TestOracleDistributedQueries.java

```diff
@@ -0,0 +1,174 @@
+        assertTableColumnNames("test_create_original", "a", "b", "c");
+
+        assertUpdate("CREATE TABLE test_create_like (LIKE test_create_original, d boolean, e varchar)");
+        assertTrue(getQueryRunner().tableExists(getSession(), "test_create_like"));
+        assertTableColumnNames("test_create_like", "a", "b", "c", "d", "e");
+
+        assertUpdate("DROP TABLE test_create_original");
+        assertFalse(getQueryRunner().tableExists(getSession(), "test_create_original"));
+
+        assertUpdate("DROP TABLE test_create_like");
+        assertFalse(getQueryRunner().tableExists(getSession(), "test_create_like"));
+    }
+
+    @Test
+    @Override
+    public void testSymbolAliasing()
+    {
+        // Replace tablename to less than 30chars, max size naming on oracle
+        String tableName = "symbol_aliasing" + System.currentTimeMillis();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT 1 foo_1, 2 foo_2_4", 1);
+        assertQuery("SELECT foo_1, foo_2_4 FROM " + tableName, "SELECT 1, 2");
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Test
+    @Override
+    public void testRenameColumn()
+    {
+        // Replace tablename to less than 30chars, max size naming on oracle
+        String tableName = "test_renamecol_" + System.currentTimeMillis();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT 'some value' x", 1);
+
+        assertUpdate("ALTER TABLE " + tableName + " RENAME COLUMN x TO y");
+        assertQuery("SELECT y FROM " + tableName, "VALUES 'some value'");
+
+        assertUpdate("ALTER TABLE " + tableName + " RENAME COLUMN y TO Z"); // 'Z' is upper-case, not delimited
+        assertQuery(
+                "SELECT z FROM " + tableName, // 'z' is lower-case, not delimited
+                "VALUES 'some value'");
+
+        // There should be exactly one column
+        assertQuery("SELECT * FROM " + tableName, "VALUES 'some value'");
+
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Test
+    @Override
+    public void testShowColumns()
+    {
+        MaterializedResult actual = computeActual("SHOW COLUMNS FROM orders");
+
+        MaterializedResult expectedParametrizedVarchar = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("orderkey", "bigint", "", "")
+                .row("custkey", "bigint", "", "")
+                .row("orderstatus", "varchar(1)", "", "")
+                .row("totalprice", "double", "", "")
+                .row("orderdate", "timestamp", "", "")
+                .row("orderpriority", "varchar(15)", "", "")
+                .row("clerk", "varchar(15)", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("comment", "varchar(79)", "", "")
+                .build();
+
+        // Until we migrate all connectors to parametrized varchar we check two options
+        assertTrue(actual.equals(expectedParametrizedVarchar),
```

Comment:
`assertEquals` and put the two parameters in one line

Reply:
changed to 1-line sanity check